diff -pur linm-0.8.1-orig/src/ncurses/main_ncurces.cpp linm-0.8.1/src/ncurses/main_ncurces.cpp
--- linm-0.8.1-orig/src/ncurses/main_ncurces.cpp	2008-02-02 20:34:33.000000000 +0900
+++ linm-0.8.1/src/ncurses/main_ncurces.cpp	2012-03-11 21:40:13.469683652 +0900
@@ -473,6 +473,8 @@ int main(int argc, char *argv[])
 	{
 		CopyConfFiles();
 				
+		g_tMainFrame.Reload();
+
 		g_tMainFrame.SetTitleChange( _bTitleChange );
 
 		if (_bMcdExe)
